graph-manager-docs/source/federation.mdx

```diff
@@ -127,9 +127,113 @@ server.listen().then(({ url }) => {
+On `apollo service:push`, the service registry writes configuration files to a cloud file system, stored securely and accessible by your API key.
+These configuration files detail which federated services are part of the graph, metadata about the federated services, including the `serviceURL`
+ specified in the `push` command, and the partial schema of that federated service. By default, when the Apollo gateway is instantiated with an
+  API key rather than a `serviceList`, it will poll for that managed config to pick up changes and smoothly roll over to the new service configuration,
+  draining in-flight requests while beginning to generate query plans for incoming requests against the new config.
+
+#### When to call `service:push`
+
+In general, when calling `service:push`, you should always roll out changes in a way that is backward compatible with existing clients to avoid downtime. Because configuration changes can affect the query planner, it's highly recommended to **call `apollo service:push` after
+all replicas of a federated service have deployed**. This way, you can be certain that only valid operations will be executable against the graph, and
+that operations will not be able to access fields or types that do not yet exist.
+
+In the rare case where a schema change is _not_ backward compatible with your gateway's query planner, you should update your service registry _before_ you deploy your federated services. In particular, schema changes that affect query planning should be done as separate changes to the registry, prior
+to other modifications. This helps avoid the scenario where the query planner generates queries that fail validation in downstream services or violate
+resolvers' expectations. Some examples of this include:
+
+1. Modifying `@key` or `@requires` dependencies while simultaneously modifying dependency fields
+2. Removing a type implementation from an interface
+
+For example, let's say we define a Channel interface in one service and define types that implement that interface in two other services:
+
+```graphql
+# Channel service
+interface Channel @key(fields: "id") {
+  id: ID!
+}
+
+# Web service
+type WebChannel implements Channel @key(fields: "id") {
+  id: ID!
+}
+
+# Email service
+type EmailChannel implements Channel @key(fields: "id") {
+  id: ID!
+}
+```
+
+To remove the `EmailChannel` implementation of the `Channel` interface:
+
+1. Register a new `email` service that removes the `EmailChannel` implementation of `Channel`.
```

Comment:
Does "register a new" actually mean "run `service:push`"?

Also let's be clear that you don't need to actually remove the EmailChannel type in the first push, just the `implements` directive? Which we should name really explicitly, not abstractly as "the implementation".

Also maybe this should actually explain the problem? The problem is that the gateway expands a query `{ getChannel { id } }` into fragmented queries right? Maybe showing more clearly would be helpful.